### ISSUE TITLE
Minor Changes for TSE

### ISSUE THREAD

**Issue by jssmith1** [DeveloperLiberationFront/iTrustInterviews#414]

**Comment by jssmith1** [comment]
- [x] p. 2, col. 1, line 45 -  when you introduce Figure 1, say here that it
is Eclipse for Java.

- [x] Section 3: For the results, are these in any particular order?
Especially 3.3.* seem to reference each other, and you jump around
from low-level to high-level and back to low level.

- [x] p. 10, col. 2 lines 8-12 - the discussion of CSRF is repetitive.

- [x] p. 12, col. 1, line 36 - typo "lead" -> "led"

- [x] section 4.3 - this reminds me of the following system, that might be
added along with the others you already mention:

Jeffrey Stylos and Brad A. Myers.  “Mica: A Programming Web-Search
Aid,” IEEE Symposium on Visual Languages and Human-Centric Computing,
VL/HCC'06, Brighton, UK, Sept 4-8, 2006. pp. 195-202.

- [x] p. 14, col. 1, line 46: "Comparing the four approaches" -- seems like
3 are listed

- [x] p. 14, col. 2, line 3: "[22], [27], [28], [42]. These three studies"
-- seems like 4

- [x] section 5.2 - need more than just a summary of these studies - you
need to specifically point out which questions and strategies confirm
what was already known, and which are different. And if you have any
that contradict previous studies, that would be especially interesting.

**Comment by jssmith1** [comment]
We tried to optimize for readability by, for example, reducing the number of substantive forward and cross references. As a result, End-User Interaction appears last in its section and categories appear in the order in which the might arise during a task (e.g., Understanding and attack->Understanding fixes->Assessing the fix). Nonetheless, some references remain and there is still some flexibility in deciding how to order the results. We have experimented with ordering the results by the number of questions in each category or number of participants who asked questions in a category, but found these orderings overemphasized the quantitative aspects of our results. We are open to suggestions from the reviewers about how to specifically improve the ordering.
